### PR TITLE
EVG-13338 Prevent editing instance types when a host is running

### PR DESCRIPTION
**Pull request by khelif96**

### CODE REVIEW

File: src/pages/spawn/spawnHost/EditSpawnHostModal.tsx [hunk 1]
```diff
@@ -178,6 +182,7 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
                 })
               }
               value={editSpawnHostState.instanceType}
+              disabled={!canEditInstanceType}
```
Comment (SupaJoon) [inline src/pages/spawn/spawnHost/EditSpawnHostModal.tsx:185]: can we conditionally render a tooltip explaining why this component is disabled?

**Review by SupaJoon — CHANGES_REQUESTED** [review]

File: src/pages/spawn/spawnHost/EditSpawnHostModal.tsx [hunk 2]
```diff
@@ -166,28 +170,37 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
...
-              placeholder="Select Instance Type"
-              onChange={(v) =>
-                dispatch({
-                  type: "editInstanceType",
-                  instanceType: v,
-                })
+            <Tooltip
+              title={
+                !canEditInstanceType
+                  ? "Cant change an instances type while it is running"
```
Comment (SupaJoon) [inline src/pages/spawn/spawnHost/EditSpawnHostModal.tsx:176]: this sentence is missing some apostrophes. can we change it to "Can't change and instance's type while it is running."

**Review by SupaJoon — APPROVED** [review]

File: src/pages/spawn/spawnHost/EditSpawnHostModal.tsx (hunk 2 above)
Comment (SupaJoon) [reply src/pages/spawn/spawnHost/EditSpawnHostModal.tsx:176]: or we can make the text more actionable: "Pause the host to adjust this field."

Comment (khelif96) [reply src/pages/spawn/spawnHost/EditSpawnHostModal.tsx:176]: Great suggestion! I had difficulty deciding on what the copy should say.

**Pull request merged**